cmd/tapcli/assets.go

```diff
@@ -261,7 +262,7 @@ var finalizeBatchCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name: feeRateName,
-			Usage: "if set, the fee rate in sat/kw to use for " +
+			Usage: "if set, the fee rate in sat/vB to use for " +
```

Comment:
Sorry, I only saw this now: but we should update the flag name to `fee_rate_sat_per_vbyte` or just `sat_per_vbyte` as well. And then update all other places where we use the `parseFeeRate` as well (for example `finalizebatch` still mentions `sat/kw` but it would now be `sat/vByte`.

Reply:
I think both callers, `finalizeBatch` and `sendAssets`, use sat/vB rn?